<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addListBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>510</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doneListBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>560</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>완료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteListBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>610</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveUpBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>670</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveDownBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>670</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="mainList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>611</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TODAY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="moveLeftBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>670</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>◀</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveRightBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>670</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
